Prace_domowe/Praca_domowa1/Grupa4/Hubert_Drążkowski/PD3_Drazkowski.ipynb

```diff
@@ -0,0 +1,918 @@
+{
```

Comment:
<p>No to jest wyższy poziom kombinowania...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>